action.py

```diff
@@ -32,16 +28,16 @@
 
 # known bad repositories that we'll just have to skip for now
 BAD_REPOS = [
-    'https://packages.monokai.pro/packages.json',
-    'https://raw.githubusercontent.com/blake-regalia/linked-data.syntaxes/master/channels/sublime/package-control.json'
+    "https://packages.monokai.pro/packages.json",
```

Comment:
Just so we don't conflict I won't push it to master now. You can remove monokai pro here, they fixed the problems.

```suggestion
```